### ISSUE TITLE
Please upgrade for angular 1.6.0 or higher

### ISSUE THREAD

**Issue by cramirez451** [cgross/angular-busy#105]
In particular the $http.success method, has been deprecated. Here is a link to all of the other breaking changes.

https://github.com/angular/angular.js/blob/master/CHANGELOG.md

**Comment by tiberiuzuld** [comment]
Made a fork with support for angular 1.6.x https://github.com/tiberiuzuld/angular-busy
Should be a drop in replacement

**Comment by geo242** [comment]
Here is a pull request for it: https://github.com/cgross/angular-busy/pull/106

**Comment by jziggas** [comment]
@tiberiuzuld not really a drop in replacement when everything is renamed

**Comment by tiberiuzuld** [comment]
@jziggas only the module name is changed. 
In rest you can continue using the old directive name and constant for configuration.

**Comment by cs-StephanieDM** [comment]
+1: please replace the $http.success method

**Comment by markusberg** [comment]
This official package at `@cgross/angular-busy` is updated for angular 1.6. Update your package.json.

**Comment by JohnMcAvinue** [comment]
It's not up on Bower though

**Comment by LauraXiulan** [comment]
Yes, please update to version 4.1.4 for Bower too, right now I cannot use it with Angular 1.6
